layouts/css/layout/_dark-theme.scss

```diff
@@ -99,7 +103,7 @@
   }
 
   footer {
-    margin-top: 0 !important;
+    // margin-top: 0 !important;
```

Comment:
Instead of commenting out, should we remove this line?

Reply:
Yes it's not necessary.